2025/lab/Keys/Week8_Binary_Outcomes.Rmd

```diff
@@ -0,0 +1,406 @@
+---
+title: "Week 8 -  Regression with Binary Outcomes"
+author: "Brent Hoagland, Lab TA"
+date: "3/24/2025"
+output:
+  pdf_document: default
+  html_document: default
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+options(scipen = 10)
+
+library(tidyverse)
+library(rstanarm)
+library(modelsummary)
+library(broom.mixed)
+options(modelsummary_get = "broom") 
+library(flextable)
+library(naniar)
+
+seed <- 1234
+```
+
+# Data Loading and Management
+
+Let's use the GSS again. This week, we'll be investigating factors associated with whether a person supports marijuana legalization. 
+
+```{r gss-loading}
+# You may need to write the code or change your working directory to properly load the GSS2022.dta file and assign it to the object gss2022
```

Comment:
I'd encourage students to ensure they are in the project, that way they shouldn't need to change the directory.